src/main/java/edu/cornell/kfs/ksr/KSRConstants.java

```diff
@@ -1,24 +1,50 @@
 package edu.cornell.kfs.ksr;
 
 public class KSRConstants {
-	public static final String KSR_DOCUMENT_MAINTAINABLE = "document.newMaintainableObject";
-	
-	public static final String KSR_NAMESPACE = "KFS-SR";
-
-	// Security Group sections
-	public static final String SECTION_SECURITY_TABS = "SecurityGroupTabs";
-
-	// Security Group properties
-	public static final String SECURITY_GROUP_NAME = "securityGroupName";
-	
-	// Security Group Tab properties
-	public static final String SECURITY_GROUP_TAB_NAME = "tabName";
-	public static final String SECURITY_GROUP_TAB_ID = "tabId";
-	public static final String SECURITY_GROUP_TAB_ORDER = "tabOrder";
-
-	// Sequence names
-	public static final String SECURITY_GROUP_SEQ_NAME = "KRSR_SEC_GRP_ID_SEQ";
-	public static final String SECURITY_GROUP_TAB_SEQ_NAME = "KRSR_SEC_GRP_TB_ID_SEQ";
-    
+    public static final String KSR_DOCUMENT_MAINTAINABLE = "document.newMaintainableObject";
+
+    public static final String KSR_NAMESPACE = "KFS-SR";
+
+    // Security Group sections
+    public static final String SECTION_SECURITY_TABS = "SecurityGroupTabs";
+
+    // Security Group parameters

```

Comment:
I realize the old code was labeling some properties as parameters by mistake, but it would still be helpful to update or remove these comments accordingly, given that the constants actually represent properties.

Also, it might be helpful to have a follow-up cleanup user story to create a KSRPropertyConstants class for storing the property-name-related values.

Reply:
Great suggestion. I updated the comments and moved the properties constants in a separate KSRPropertyConstants class.